src/actions/bookshelf/serializers/flat.js

```diff
@@ -0,0 +1,141 @@
+import objectUtils from '../../../utils/object-utils'
+
+const debug = false
+
+// -----------------------------------------------------------------------------
+// Serializer for "flattened" output format.
+//
+// NOTE: The resource model type is set as "type".
+// -----------------------------------------------------------------------------
+export default function serialize (type, data, joint) {
+  let json = {}
+
+  if (data) {
+    if (debug) console.log(`[Serializer] Serialize ${type} data to flattened format =>`, data)
+
+    // Build data package
+    const packageType = (data.attributes) ? 'item' : 'collection'
+    if (packageType === 'item') {
+      json = buildItemPackage(type, data, joint)
+    } else {
+      json = buildCollectionPackage(type, data, joint)
+    }
+  }
+
+  return json
```

Comment:
A nitpick suggestion on coding style. It seems cognitively lighter to not have to wrap the code in a 2-layer if-else statement. Same idea would go a long way if applied to the rest of the code to reduce a lot of the indentation.

```suggestion
  return !data
    ? {}
    : packageType === 'item'
    ? buildItemPackage(type, data, joint)
    : buildCollectionPackage(type, data, joint)
```

Reply:
Yep! Perfect. It was copied directly from JSON API serializer, and I just molded it down. You are right.